main.py

```diff
@@ -130,6 +130,17 @@ def get_args():
     )
     parser.set_defaults(deterministic_agents=False)
 
+    parser.add_argument(
+        "-l",
+        "--log_level",
+        default="info",
+        type=str,
+        required=False,
+        help="sets the log_level. it must be one of {{{}}}.".format(
+            ", ".join(HUMAN_LOG_LEVELS)
+        ),
```

Comment:
Why have `{{{}}}` instead of just `{}`?

Reply:
I just thought it would be nice to show the possible options as a set, but I'll just list them.